src/Route/RouteCollectionInterface.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Route;
+
+use Symfony\Component\Routing\Route;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+interface RouteCollectionInterface
```

Comment:
Maybe this should go on 3.x

Reply:
No a big need, since multiple signature use `RouteCollection` and changing the typehint is BC.

Reply:
Then I leave it as is.